src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,70 @@
-export const Tabs = () => {};
+import { FC } from 'react';
+
+import cn from 'classnames';
+
+import { Tab } from '../../types';
+
+type Props = {
+  onTabSelected: (tabNr: Tab) => void,
+  tabs: Tab[] | [],
```

Comment:
`tabs` will always be of type `Tab[]`
```suggestion
  tabs: Tab[],
```